frontend/src/libs/types/button-style.type.ts

```diff
@@ -7,6 +7,7 @@ type ButtonStyle =
   | 'play-button'
   | 'active'
   | 'start-button'
+  | 'primary-blue'
```

Comment:
Maybe we can pass color directly and stay with existing primary

Reply:
we can try, but we should also consider hover and disabled states. The color of the states should also be changed. What do you think? Or do you want to add a color prop that would be added to classNames and then change button styles depending on color?

Reply:
Not sure whether the naming is suitable

Reply:
maybe I can add a prop like `primaryStyleColor`? It would be a union with colors like: `default | blue`. Depending on this prop we would extend styles for button

Reply:
Good point, but let's without primary: `styleColor`